pkg/operator/reports/name.go

```diff
@@ -0,0 +1,36 @@
+package reports
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aquasecurity/trivy-operator/pkg/kube"
+	"k8s.io/apimachinery/pkg/util/validation"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func NameFromController(controller client.Object) string {
+	kind := controller.GetObjectKind().GroupVersionKind().Kind
+	name := controller.GetName()
+	reportName := fmt.Sprintf("%s-%s", strings.ToLower(kind), name)
+	if isValidName(reportName) {
+		return reportName
+	}
+	return fmt.Sprintf("%s-%s", strings.ToLower(kind), kube.ComputeHash(name))
+}
+
+func NameFromControllerContainer(controller client.Object, container string) string {
```

Comment:
I would change the func name to be more clear to `ControllerContainerName`

Reply:
This name I also didn't like, though I like the consistency with the other method. When I was looking, I actually thought it could have been `NameFromControllerAndContainer`. Though, I would keep it closer to whatever is decided on for the method above.